app/frontend/src/components/DateFormats.tsx

```diff
@@ -0,0 +1,212 @@
+export const dateFormats = {
+  "af-ZA": "YYYY/MM/DD",
```

Comment:
Where did you get this information from? I'm wondering if this is included in some date time library we're using (`dayjs`?) and so we don't need to maintain a manual mapping like this ourselves.

Reply:
Would this [localised format suggestion in the docs](https://day.js.org/docs/en/display/format#localized-formats) work?

Reply:
Any luck with this @angelika7 ?